packages/terra-form-select/src/Combobox.jsx

```diff
@@ -237,6 +242,7 @@ class Combobox extends React.Component {
         clearOptionDisplay={clearOptionDisplay}
         inputId={inputId}
         resetComboboxValue={this.handleResetComboboxValue}
+        ariaLabel={ariaLabel}
```

Comment:
```suggestion
        ariaLabel={this.ariaLabel()}
```
I think you should use ariaLabel() function  here instead of assigning with direct prop.